src/main/resources/application.properties

```diff
@@ -1 +1,4 @@
 spring.application.name=GitHubAiAPI
```

Comment:
Please ensure that the property key 'spring.application.name' is not removed as it is essential for identifying the application in a distributed setup.